README.md

```diff
@@ -12,19 +12,7 @@ A Python Client and CLI tool for Monday.com
    * [Changing Column Values](#changing-column-values)
```

Comment:
Please do not include this change as part of this PR.  Please do so for the "Github Wiki - Move Entity And Column Values Documentation" task.